sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs

```diff
@@ -135,20 +137,156 @@ public virtual Response<AnalyzeConversationTaskResult> AnalyzeConversation(strin
             options ??= new AnalyzeConversationOptions(textConversationItem);
             CustomConversationalTask customConversationalTask = new CustomConversationalTask(options, customConversationTaskParameters);
 
-            using DiagnosticScope scope = Diagnostics.CreateScope($"{nameof(ConversationAnalysisClient)}.{nameof(AnalyzeConversation)}");
+            using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(ConversationAnalysisClient)}.{nameof(AnalyzeConversation)}External");
             scope.AddAttribute("projectName", project.ProjectName);
             scope.AddAttribute("deploymentName", project.DeploymentName);
             scope.Start();
 
             try
             {
-                return _analysisRestClient.AnalyzeConversation(customConversationalTask, cancellationToken);
+                Utf8JsonRequestContent content = new Utf8JsonRequestContent();
+                content.JsonWriter.WriteObjectValue(customConversationalTask);
+
+                Response response = AnalyzeConversation(content, new RequestContext() { CancellationToken = cancellationToken });
+
+                switch (response.Status)
+                {
+                    case 200:
+                        {
+                            AnalyzeConversationTaskResult value = default;
+                            using var document = JsonDocument.Parse(response.ContentStream);
+                            value = AnalyzeConversationTaskResult.DeserializeAnalyzeConversationTaskResult(document.RootElement);
+                            return Response.FromValue(value, response);
+                        }
+                    default:
+                        throw ClientDiagnostics.CreateRequestFailedException(response);
+                }
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>Analyzes a conversational utterance.</summary>
+        /// <param name="input">The <see cref="GeneratedConversation"/> used for tasks input.</param>
+        /// <param name="tasks"> <see cref="AnalyzeConversationLROTask"/> defines the tasks to be used.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> to cancel the request.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="input"/> or <paramref name="tasks"/> or is null.</exception>
+        /// <exception cref="RequestFailedException">The service returned an error. The exception contains details of the service error.</exception>
+        public virtual async Task<Operation<AnalyzeConversationJobState>> StartAnalyzeConversationAsync(IEnumerable<GeneratedConversation> input, IEnumerable<AnalyzeConversationLROTask> tasks, CancellationToken cancellationToken = default)
```

Comment:
instead of letting the user deal with `BinaryData` as a return type, I created this so we can modify the return type to the needed one. also, it will be easier for the user to pass the parameters directly without needing to convert it to content.

Reply:
I appreciate that, but why would it be `BinaryData` in this case? Shouldn't all our inference models have associated classes?

Reply:
If you checked the generated `ConversationAnalysisClient.cs` you will find that `SubmitJob` function returns BinaryData

Reply:
Do we know why? That seems like a broken code generation scenario. @ShivangiReja could you explain?

Reply:
Or, @ShivangiReja, is this just because it's a DPG? If that's the case, @ZulaMostafa, the way I planed to do this for the 1.0.0 GA was that we'll generate LLC models for the DPG, and will take/return those models as-is for LLC-like convenience methods. Basically, the idea is to ship an LLC with DPG protocol methods.

Reply:
[ SubmitJob](https://github.com/Azure/azure-sdk-for-net/blob/1bced6de0c38dcd5ef522207c9ce00466903718a/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs#L353) is a DPG method, that's why it is returning BinaryData.

Reply:
So what we should have is an LLC-like method that takes the correct input model and returns the correct output model - almost like a DPG. We can add convenience methods like this later. But if the goal is to not have customers parse a `BinaryData` - with which I complete empathize - the "LLC convenience" method provides the most flexibility.

Reply:
@heaths IMO a combination between LLC and DPG would be better for .NET. 
Taking the models generation from LLC, and client layer from DPG since DPG wraps LRO operation handling. We will still need to add a convenience layer to match the needed input/output but still better than bothering to handle LROs ourselves.

Reply:
Given offline discussion, we'll merge as-is for now but will need to resolve this by 1.1.0 GA.

Reply:
I've opened https://github.com/Azure/azure-sdk-for-net/issues/28936 to track.